runtime/integration-tests/src/integrations_mock/test_manta.rs

```diff
@@ -198,6 +198,7 @@ fn reward_fees_to_block_author_and_treasury() {
             (ALICE.clone(), INITIAL_BALANCE),
             (BOB.clone(), INITIAL_BALANCE),
             (CHARLIE.clone(), INITIAL_BALANCE),
+            (Treasury::account_id(), INITIAL_BALANCE),
```

Comment:
wonder why this wasn't failing for the Calamari test

Reply:
The existential deposit is lower 0.1 KMA compared to 1 MANTA.